### ISSUE TITLE
Project is dead

### ISSUE THREAD

**Issue by last-partizan** [y10h/pytils#36]
Судя по всему проект мертв, и поддержка python3 оставляет желать лучшего?

Есть какие-то альтернативы?

**Comment by y10h** [comment]
Да, проект скорее мертв чем жив. Если интересна поддержка/адаптация, то я передам владение проектом.

**Comment by last-partizan** [comment]
Можно попробовать, я пока смотрел - все что нам нужно работает на python3 уже, но может и починю что, да и пулл-реквесты поразгребать надо.

**Comment by last-partizan** [comment]
ping
